cmd/storj/LICENSE

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
I am especially anxious about this. for the client code specifically, we will want large enterprise customers to be able to link our code into their binaries, and some large enterprise customers have a very strict no copyleft rule.

further, there's two things the LGPL doesn't have that apache v2 fixes. first: apachev2 explicitly says that if a developer contributes code to the project, they are agreeing to license their code under the same license. that's assumed but not actually the case for many other licenses. that's important, especially because point two: apachev2 has an explicit patent grant clause. if the copyright holder owns patents on the code they're submitting, they release the users of the open source project from any threat of patent infringement violation.

so, not saying LGPL is out, certainly, but i think we need to talk about it, especially for the client code.

Reply:
I am not a fanatic about licenses. I was just following the established pattern for Storj client code (from libstorj and bindings).

It would be best to have this license discussion ASAP as we are already committing code to this repo without any license at all, which is the worst possible choice.